app/code/Codifi/Sales/ViewModel/Order/View/Info.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Codifi_Sales
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+
+namespace Codifi\Sales\ViewModel\Order\View;
+
+use Codifi\Sales\Model\Source\OrderType;
+use Magento\Framework\View\Element\Block\ArgumentInterface;
+use Magento\Sales\Model\Order;
+use Magento\Framework\Exception\LocalizedException;
+
+/**
+ * Class Info
+ * @package Codifi\Sales\ViewModel\Order\View
+ */
+class Info implements ArgumentInterface
+{
+    /**
+     * Order type options
+     *
+     * @var OrderType
+     */
+    private $orderTypeSource;
+
+    /**
+     * Info constructor.
+     *
+     * @param OrderType $orderTypeSource
+     */
+    public function __construct(
+        OrderType $orderTypeSource
+    ){
+        $this->orderTypeSource = $orderTypeSource;
+    }
+
+
+    /**
+     * Get order_type attrubute label
+     *
+     * @param Order $currentOrder
+     * @return string
+     * @throws LocalizedException
+     */
+    public function getAttributeLabel($currentOrder): string
+    {
+        $response = '';
+        $orderType = $currentOrder->getData('order_type');
+        $options = $this->orderTypeSource->getAllOptions();
+        foreach ($options as $item){
+            if ($item['value'] === $orderType) {
+                $response = $item['label'];
+            }
+        }
+
+        return $response;
```

Comment:
1. In case if you don't find label, it's better to display field value than empty string
2. Variable name doesn't its purpose. Name should be `$label` or `$attributeLabel`